Misc/NEWS.d/next/Core and Builtins/2018-12-22-22-19-51.bpo-35560.9vMWSP.rst

```diff
@@ -0,0 +1,2 @@
+Removed incorrect assert statement that caused segfault in debug builds.
```

Comment:
This is too general and unlikely useful. Please specify place/conditions of the segfault.

Reply:
Added a note on the function and assert statement. Thanks.